test/fixtures/app/gulpfile.js

```diff
@@ -15,16 +14,23 @@ gulp.task('build', ['clean'], function(){
   return gulp.src(['src/**'], { cwd: __dirname, base: path.join(__dirname, 'src') })
     .pipe(gulp.dest('build', { cwd: __dirname }));
 });
-
+var localePath = path.join(__dirname, 'locale');
+var installDepLocalePath = path.join(__dirname, 'installed_deps/locale');
 var options = {
-  localeDir: path.join(__dirname, 'locale')
+  localeDirs: [localePath, installDepLocalePath]
 };
 
-gulp.task('translate', ['build'], function(){
+gulp.task('translateMultidirs', ['build'], function(){
   return gulp.src(['build'], { cwd: __dirname })
     .pipe(statici18n(options));
 });
 
-gulp.task('default', ['translate']);
+gulp.task('translatesingledir', ['build'], function(){
```

Comment:
The name changes to the tasks here don't make much sense to me.

What about keeping `translate` unchanges and renaming this one `translate-with-deps`